infrastructure/src/main/java/org/corfudb/infrastructure/management/FailureDetector.java

```diff
@@ -93,7 +94,7 @@ public PollReport poll(
 
         // Perform polling of all responsive servers.
         return pollRound(
-                layout.getEpoch(), allServers, routerMap, sequencerMetrics,
+                layout.getEpoch(), layout.getClusterId(), allServers, routerMap, sequencerMetrics,
```

Comment:
Just pass the whole Layout?

Reply:
I feel like it is sufficient enough for the poll round to perform iterations. I would extract servers, epoch and clusterId in a separate DTO, for example, PollData but it's not enough the effort, since it's used only in two functions.